### DISCUSSION TITLE
Labels for form card components

### DISCUSSION THREAD

**Discussion by LemonPie22** [filamentphp/filament#5139] [Help] [answered]
I have some card components on a form:

```
Forms\Components\Card::make()
    ->schema(
        [
            Forms\Components\TextInput::make('dimension_x'),
            Forms\Components\TextInput::make('dimension_y'),
            Forms\Components\TextInput::make('dimension_z'),
            Forms\Components\Select::make('dimension_unit_id')
                ->relationship('dimensionUnit', 'plural_name'),
        ]
    )->columns(4),
Forms\Components\Card::make()
    ->schema(
        [
            Forms\Components\TextInput::make(
                'net_weight'
            ),
            Forms\Components\TextInput::make(
                'gross_weight'
            ),
            Forms\Components\Select::make('weight_unit_id')
                ->relationship('weightUnit', 'plural_name'),
        ]
    )->columns(3),
```

The result is as follows:

![Screenshot 2022-12-01 at 23 38 39](https://user-images.githubusercontent.com/107488019/205181940-782163e2-c75f-4751-8676-041b6ea538b4.png)

Is there a way to add labels (and maybe descriptions) to these cards. I'm looking for something like this:

![Screenshot 2022-12-01 at 23 30 54](https://user-images.githubusercontent.com/107488019/205182077-83064ed5-b67c-42d2-bc68-566cd4d63209.png)

**Comment by danharrin** [comment, marked as answer]
I would suggest using a Fieldset or a Section instead?

Fieldsets have a transparent background and a smaller label.

Sections have a heading and subheading and are collapsible and have a background

**Reply by LemonPie22** [reply]
Thank you @danharrin. I've switched to Sections and they give me exactly what I need.
